pepper-apis/config/application.conf.ctmpl

```diff
@@ -93,7 +93,7 @@
     {{if eq $environment "dev"}}
         "baseTestUrl": "https://pepper-dev.datadonationplatform.org",
         "baseWebDriverUrl": "https://basil-dev.datadonationplatform.org",
-        "dsmBaseUrl": "https://dsm-dev.datadonationplatform.org:443"
+        "dsmBaseUrl": "https://10.128.0.6:443"
```

Comment:
Now-static internal IP for dev DSM.  Would apply this pattern across all envs.

Reply:
Even if it's static, why is the URL using the IP directly and not sticking with the domain name?